apis/r/R/TileDBGroup.R

```diff
@@ -195,10 +212,8 @@ TileDBGroup <- R6::R6Class(
       stopifnot(is_scalar_character(name))
       private$check_open_for_write()
 
-      tiledb::tiledb_group_remove_member(
-        grp = private$.tiledb_group,
-        uri = name
-      )
+      #tiledb::tiledb_group_remove_member(grp = private$.tiledb_group, uri = name)
+      c_group_remove_member(private$.tiledb_group, name)
```

Comment:
Can we clean out the commented code?
```suggestion
      c_group_remove_member(private$.tiledb_group, name)
```